azure-pipelines.yml

```diff
@@ -798,6 +811,22 @@ jobs:
                    $(Slack.team-daml-ci)
           }
 
+          tell_daml() {
+              local message
+              message=$1
+              curl -XPOST \
+                   -i \
+                   -H 'Content-Type: application/json' \
+                   --data "{\"text\":\"<https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for release PR <https://github.com/digital-asset/daml/pull/$(pr.num)|#$(pr.num)> has completed with status ${message}.\"}" \
+                   $(Slack.team-daml)
+          }
+
+          if [ "$(is_release)" == "true" ]; then
```

Comment:
Does this trigger on both the PR and on master?

Reply:
No, the `notify_user` job never runs on master.